WordPress/Classes/ViewRelated/Suggestions/SuggestionsTableView.m

```diff
@@ -29,7 +32,7 @@ - (instancetype)initWithSiteID:(NSNumber *)siteID
         _suggestions = [[SuggestionService sharedInstance] suggestionsForSiteID:_siteID];
         _searchText = @"";
         _searchResults = [[NSMutableArray alloc] init];
-        
+        [self setUseTransparentHeader:NO];
```

Comment:
It's a good practise not to use setters in the initializer. Reason why is: this `setUseTransparentHeader` ends up updating the `self.headerView`'s backgroundColor.

(Which in this case, at this point, hasn't been alloc'ed yet)